.github/workflows/build-docker-images-tags.yml

```diff
@@ -0,0 +1,91 @@
+---
+name: Build images for tags
+
+on:
+  push:
+    tags:
+    - 'auth-*'
+    - 'dnsdist-*'
+    - 'rec-*'
```

Comment:
Pretty much the same reasoning, there is no valid reason to push a non `dnsdist-` tag to a stable dnsdist branch so it should be fine.